src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -265,7 +265,7 @@ private static Event GetEvent(LogEvent e)
 				Id = e.TryGetScalarString(SpecialKeys.ActionId, out var actionId)
 					? actionId
 					: null,
-				Kind = e.TryGetScalarString(SpecialKeys.ActionKind, out var actionKind) ? actionKind : null,
+				Kind = e.TryGetScalarString(SpecialKeys.ActionKind, out var actionKind) ? actionKind : "event",
```

Comment:
You can add a Serilog-enricher that always inserts the ActionKind-property in your own solution. What is good default for you, might be noise for others.